src/utils/components/DiskModal/components/DiskInterfaceSelect/DiskInterfaceSelect.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, useEffect } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import FormGroupHelperText from '@kubevirt-utils/components/FormGroupHelperText/FormGroupHelperText';
+import FormPFSelect from '@kubevirt-utils/components/FormPFSelect/FormPFSelect';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { diskTypes } from '@kubevirt-utils/resources/vm/utils/disk/constants';
+import { FormGroup, SelectOption } from '@patternfly/react-core';
+
+import { DiskFormState, InterfaceTypes } from '../../utils/types';
+
+import { diskInterfaceOptions } from './utils/constants';
+
+type DiskInterfaceSelectProps = {
+  isVMRunning: boolean;
+};
+
+const DiskInterfaceSelect: FC<DiskInterfaceSelectProps> = ({ isVMRunning }) => {
+  const { t } = useKubevirtTranslation();
+  const { control, setValue, watch } = useFormContext<DiskFormState>();
+  const [diskType, diskInterface] = watch(['diskType', 'diskInterface']);
+
+  useEffect(() => {
+    // only SCSI is supported for hotplug
+    if (isVMRunning && diskInterface !== InterfaceTypes.SCSI) {
```

Comment:
shouldnt this be the only option if VM is running? I'm guessing u are changing for the user without his knowledge with this effect?

Reply:
you guessed correctly, I changed default value of interface based on a running VM status